### ISSUE TITLE
Add code to resize and otherwise adjust images uploaded via the admin

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-server#14475]
Supports https://github.com/mozilla/addons-server/issues/14471
Depends on #14472

The code that uploads images likely needs to do some manipulation of those images in order to ensure they are of a reasonable size and are consistent. This might just be resizing, but perhaps other things will need to be done too. There was also discussion of generating thumbnails for the images at the same time.

Specific dimensions desired for the images and thumbnails can be added to this issue.

**Comment by ioanarusiczki** [comment]
@xlisachan I verified on admin dev with FF78(Win10):
- the thumbnails from Primary Hero can be uploaded, the thumbnails have a fix width of 150 px and a variable height. 
- the thumbnails from discovery items have 150px x 120px.
- maybe I should see the 960x640 dimension on the frontend with View Image Info (probably that's incoming in another issue). I attached the images for some add-ons from discovery items but the frontend does not display them yet (only the previous images from "Image" can be attached).

**Comment by xlisachan** [comment]
@ioanarusiczki, thanks for verifying! You are correct, the frontend should not display the 960x640 dimension images yet, as that will be covered in another issue. This pull request only resizes the images that are uploaded to two dimensions. Thank you!
